backend/src/services/linear/api/resolvers.py

```
@@ -10673,25 +10673,71 @@ def resolve_issueUpdate(obj, info, **kwargs):
+            # This ensures the join table is properly maintained
+            if label_ids:
+                # Fetch all IssueLabel objects for the given IDs
+                new_labels = (
+                    session.query(IssueLabel).filter(IssueLabel.id.in_(label_ids)).all()
+                )
+
+                # Validate that all label IDs exist
+                found_ids = {label.id for label in new_labels}
+                missing_ids = set(label_ids) - found_ids
+                if missing_ids:
+                    raise Exception(f"Label(s) not found: {', '.join(missing_ids)}")
```

Ruff (0.14.4)

Abstract `raise` to an inner function

(TRY301)